<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceImport</class>
 <widget class="QWidget" name="DeviceImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт устройств</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/32x32/import/ico/import_32x32.png</normaloff>:/32x32/import/ico/import_32x32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Начальные данные</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Файл импорта:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditTwoButton" name="fileForImport">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="importFileStatus">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons.qrc">:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Фирма:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="organization">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="organizationStatus">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons.qrc">:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Состояние устройств:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="deviceState">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonDeviceStateEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/16x16/book/ico/book_open_16x16.png</normaloff>:/16x16/book/ico/book_open_16x16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="deviceStateStatus">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons.qrc">:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Состояние лицензий:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="licenseeState">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonLicenseeStateEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/16x16/book/ico/book_open_16x16.png</normaloff>:/16x16/book/ico/book_open_16x16.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="licenseeStateStatus">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons.qrc">:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</pixmap>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Филиал/Представительство:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="filials">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Сопоставления</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="buttonComparisonWP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сопоставление рабочих мест</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonComparisonTypeDevice">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сопоставление типов устройств</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonComparisonTypeLicense">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сопоставление типов лицензий</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="buttonComparisonProducer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сопоставление производителей</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonImport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Импорт</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/32x32/import/ico/import_32x32.png</normaloff>:/32x32/import/ico/import_32x32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditTwoButton</class>
   <extends>QLineEdit</extends>
   <header>plugins/lineedittwobutton/lineedittwobutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>organization</tabstop>
  <tabstop>deviceState</tabstop>
  <tabstop>buttonDeviceStateEdit</tabstop>
  <tabstop>licenseeState</tabstop>
  <tabstop>buttonLicenseeStateEdit</tabstop>
  <tabstop>filials</tabstop>
  <tabstop>buttonComparisonWP</tabstop>
  <tabstop>buttonComparisonTypeDevice</tabstop>
  <tabstop>buttonImport</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
